app/(auth)/login/LoginForm.tsx

```diff
@@ -0,0 +1,77 @@
+'use client';
```

Comment:
Let's conform to our usual pattern of leaving an empty line between directives and imports

(in all files)

Reply:
e3c9ecd5cfb406aea62caf83b697ddaf901af81c